Public/Src/Pips/Dll/Operations/Process.cs

```diff
@@ -329,7 +338,8 @@ public sealed partial class Process : Pip
             TimeSpan? nestedProcessTerminationTimeout = null,
             AbsentPathProbeInUndeclaredOpaquesMode absentPathProbeMode = AbsentPathProbeInUndeclaredOpaquesMode.Unsafe,
             DoubleWritePolicy doubleWritePolicy = DoubleWritePolicy.DoubleWritesAreErrors,
-            ContainerIsolationLevel containerIsolationLevel = ContainerIsolationLevel.None)
+            ContainerIsolationLevel containerIsolationLevel = ContainerIsolationLevel.None,
+            int weight = 1)

```

Comment:
> = 1) [](start = 22, length = 5)

We have the default coded in too many places...

Can we make this a required field -or- nullable?